src/applications/hca/containers/ConfirmationPageNew.jsx

```diff
@@ -0,0 +1,178 @@
+import React from 'react';
+import moment from 'moment';
+import { connect } from 'react-redux';
+import Scroll from 'react-scroll';
+
+import { focusElement } from '../../../platform/utilities/ui';
```

Comment:
nit: this can be cleaned up to be just `/platform/utilities/ui` without the `../../../`

Reply:
When I deleted '../../..', eslint barked about "import/no-absolute-path" rule:
![import-change-eslint-error](https://user-images.githubusercontent.com/587583/56446997-b87ab780-62ba-11e9-90ca-263bedf9363b.png)

I'll just leave the relative path alone.

Reply:
Ah, yeah totally optional. I also misled you. Gotta delete ../../../ including the final slash. ESLint didn't like the import path to start with a slash 😛